ansible/vagrant/Vagrantfile

```diff
@@ -137,6 +137,10 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
       ansible.groups = groups
       ansible.playbook = "../cluster.yml"
       ansible.limit = "all" #otherwise the metadata wont be there for ipv4?
+      # Example variables which we might set.
+      #		iface: This is for binding flannel to the non-default eth device.
+      #		apiserver_runtime_config: This can be provided to, for example, enable deployments (hpa).
+      # ansible.extra_vars = { flannel_iface: "eth1", apiserver_runtime_config: "extensions/v1beta1/deployments=true" }
```

Comment:
yeah thats a good point, we could put it in the params. 
ill do that instead and move it out of Vagrantfile.
will update...